2020-46-20 12:46:22-__init__-DEBUG-Using selector: SelectSelector
2020-46-20 12:46:25-__init__-DEBUG-Using selector: SelectSelector
2020-08-20 14:08:32-trigger_reload-INFO-D:\Study\School\e2e\server\backend\models.py changed, reloading.
2020-08-20 14:08:32-__init__-DEBUG-Using selector: SelectSelector
2020-08-20 14:08:36-__init__-DEBUG-Using selector: SelectSelector
2020-08-20 14:08:42-__init__-DEBUG-Using selector: SelectSelector
2020-08-20 14:08:54-__init__-DEBUG-Using selector: SelectSelector
2020-20-20 14:20:13-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-20-20 14:20:13-__init__-DEBUG-Using selector: SelectSelector
2020-20-20 14:20:17-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-20-20 14:20:17-__init__-DEBUG-Using selector: SelectSelector
2020-21-20 14:21:44-trigger_reload-INFO-D:\Study\School\e2e\server\server\settings.py changed, reloading.
2020-21-20 14:21:44-__init__-DEBUG-Using selector: SelectSelector
2020-51-20 14:51:09-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 126, in messageDetail
    loginingUserid = int(request.COOKIES["loginingUserid"])
KeyError: 'loginingUserid'
2020-51-20 14:51:53-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 126, in messageDetail
    loginingUserid = int(request.COOKIES["loginingUserid"])
KeyError: 'loginingUserid'
2020-57-20 14:57:45-trigger_reload-INFO-D:\Study\School\e2e\server\server\settings.py changed, reloading.
2020-57-20 14:57:45-__init__-DEBUG-Using selector: SelectSelector
2020-58-20 14:58:22-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 126, in messageDetail
    loginingUserid = int(request.COOKIES["loginingUserid"])
KeyError: 'loginingUserid'
2020-58-20 14:58:48-trigger_reload-INFO-D:\Study\School\e2e\server\server\settings.py changed, reloading.
2020-58-20 14:58:48-__init__-DEBUG-Using selector: SelectSelector
2020-58-20 14:58:57-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 126, in messageDetail
    loginingUserid = int(request.COOKIES["loginingUserid"])
KeyError: 'loginingUserid'
2020-20-20 15:20:00-trigger_reload-INFO-D:\Study\School\e2e\server\server\settings.py changed, reloading.
2020-20-20 15:20:00-__init__-DEBUG-Using selector: SelectSelector
2020-21-20 15:21:09-trigger_reload-INFO-D:\Study\School\e2e\server\server\settings.py changed, reloading.
2020-21-20 15:21:09-__init__-DEBUG-Using selector: SelectSelector
2020-21-20 15:21:11-trigger_reload-INFO-D:\Study\School\e2e\server\server\settings.py changed, reloading.
2020-21-20 15:21:11-__init__-DEBUG-Using selector: SelectSelector
2020-21-20 15:21:23-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 126, in messageDetail
    loginingUserid = int(request.COOKIES["loginingUserid"])
KeyError: 'loginingUserid'
2020-21-20 15:21:55-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 126, in messageDetail
    loginingUserid = int(request.COOKIES["loginingUserid"])
KeyError: 'loginingUserid'
2020-24-20 15:24:39-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-24-20 15:24:39-__init__-DEBUG-Using selector: SelectSelector
2020-24-20 15:24:52-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-24-20 15:24:52-__init__-DEBUG-Using selector: SelectSelector
2020-24-20 15:24:58-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-24-20 15:24:58-__init__-DEBUG-Using selector: SelectSelector
2020-25-20 15:25:00-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-25-20 15:25:00-__init__-DEBUG-Using selector: SelectSelector
2020-25-20 15:25:06-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-25-20 15:25:06-__init__-DEBUG-Using selector: SelectSelector
2020-25-20 15:25:23-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 126, in messageDetail
    getData = json.loads(request.body)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2020-25-20 15:25:51-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-25-20 15:25:51-__init__-DEBUG-Using selector: SelectSelector
2020-25-20 15:25:54-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-25-20 15:25:54-__init__-DEBUG-Using selector: SelectSelector
2020-25-20 15:25:57-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-25-20 15:25:57-__init__-DEBUG-Using selector: SelectSelector
2020-26-20 15:26:08-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-26-20 15:26:08-__init__-DEBUG-Using selector: SelectSelector
2020-26-20 15:26:16-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 127, in messageDetail
    getData = json.loads(request.body)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2020-26-20 15:26:25-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-26-20 15:26:25-__init__-DEBUG-Using selector: SelectSelector
2020-26-20 15:26:31-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 127, in messageDetail
    getData = json.loads(request.body)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2020-28-20 15:28:03-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-28-20 15:28:03-__init__-DEBUG-Using selector: SelectSelector
2020-28-20 15:28:13-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 126, in messageDetail
    loginingUserid = int(request.COOKIES["loginingUserid"])
KeyError: 'loginingUserid'
2020-28-20 15:28:54-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-28-20 15:28:54-__init__-DEBUG-Using selector: SelectSelector
2020-28-20 15:28:59-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-28-20 15:28:59-__init__-DEBUG-Using selector: SelectSelector
2020-30-20 15:30:35-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 126, in messageDetail
    loginingUserid = int(request.COOKIES["loginingUserid"])
KeyError: 'loginingUserid'
2020-32-20 15:32:35-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-32-20 15:32:35-__init__-DEBUG-Using selector: SelectSelector
2020-32-20 15:32:43-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-32-20 15:32:43-__init__-DEBUG-Using selector: SelectSelector
2020-32-20 15:32:49-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 112, in messageDetail
    loginingUserid = int(request.COOKIES["loginingUserid"])
KeyError: 'loginingUserid'
2020-33-20 15:33:13-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-33-20 15:33:13-__init__-DEBUG-Using selector: SelectSelector
2020-33-20 15:33:36-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 126, in messageDetail
    loginingUserid = int(request.COOKIES["loginingUserid"])
KeyError: 'loginingUserid'
2020-35-20 15:35:16-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-35-20 15:35:16-__init__-DEBUG-Using selector: SelectSelector
2020-35-20 15:35:30-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-35-20 15:35:30-__init__-DEBUG-Using selector: SelectSelector
2020-35-20 15:35:35-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-35-20 15:35:35-__init__-DEBUG-Using selector: SelectSelector
2020-35-20 15:35:38-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-35-20 15:35:38-__init__-DEBUG-Using selector: SelectSelector
2020-36-20 15:36:46-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-36-20 15:36:46-__init__-DEBUG-Using selector: SelectSelector
2020-36-20 15:36:51-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-36-20 15:36:51-__init__-DEBUG-Using selector: SelectSelector
2020-37-20 15:37:23-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 127, in messageDetail
    loginingUserid = int(request.session["loginingUserid"])
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\contrib\sessions\backends\base.py", line 64, in __getitem__
    return self._session[key]
KeyError: 'loginingUserid'
2020-37-20 15:37:53-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-37-20 15:37:53-__init__-DEBUG-Using selector: SelectSelector
2020-39-20 15:39:25-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-39-20 15:39:25-__init__-DEBUG-Using selector: SelectSelector
2020-39-20 15:39:41-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-39-20 15:39:41-__init__-DEBUG-Using selector: SelectSelector
2020-39-20 15:39:43-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-39-20 15:39:43-__init__-DEBUG-Using selector: SelectSelector
2020-39-20 15:39:51-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-39-20 15:39:51-__init__-DEBUG-Using selector: SelectSelector
2020-39-20 15:39:54-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-39-20 15:39:54-__init__-DEBUG-Using selector: SelectSelector
2020-40-20 15:40:07-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-40-20 15:40:07-__init__-DEBUG-Using selector: SelectSelector
2020-40-20 15:40:16-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-40-20 15:40:16-__init__-DEBUG-Using selector: SelectSelector
2020-40-20 15:40:40-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-40-20 15:40:40-__init__-DEBUG-Using selector: SelectSelector
2020-40-20 15:40:43-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-40-20 15:40:43-__init__-DEBUG-Using selector: SelectSelector
2020-40-20 15:40:47-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-40-20 15:40:47-__init__-DEBUG-Using selector: SelectSelector
2020-40-20 15:40:49-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-40-20 15:40:49-__init__-DEBUG-Using selector: SelectSelector
2020-40-20 15:40:52-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-40-20 15:40:52-__init__-DEBUG-Using selector: SelectSelector
2020-41-20 15:41:12-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-41-20 15:41:12-__init__-DEBUG-Using selector: SelectSelector
2020-41-20 15:41:14-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-41-20 15:41:14-__init__-DEBUG-Using selector: SelectSelector
2020-41-20 15:41:17-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-41-20 15:41:17-__init__-DEBUG-Using selector: SelectSelector
2020-41-20 15:41:40-log_response-ERROR-Internal Server Error: /apis/user/18138754
Traceback (most recent call last):
  File "D:\Study\School\e2e\server\backend\views.py", line 76, in user_detail
    user_temp = user.objects.get(userid=pk)
UnboundLocalError: local variable 'user' referenced before assignment

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 77, in user_detail
    except user.DoesNotExist:
UnboundLocalError: local variable 'user' referenced before assignment
2020-42-20 15:42:07-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-42-20 15:42:07-__init__-DEBUG-Using selector: SelectSelector
2020-42-20 15:42:35-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-42-20 15:42:35-__init__-DEBUG-Using selector: SelectSelector
2020-42-20 15:42:38-log_response-ERROR-Internal Server Error: /apis/user/18138754
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 86, in user_detail
    login(request, user)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\contrib\auth\__init__.py", line 131, in login
    user_logged_in.send(sender=user.__class__, request=request, user=user)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\dispatch\dispatcher.py", line 175, in send
    for receiver in self._live_receivers(sender)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\dispatch\dispatcher.py", line 175, in <listcomp>
    for receiver in self._live_receivers(sender)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\contrib\auth\models.py", line 20, in update_last_login
    user.save(update_fields=['last_login'])
TypeError: save() missing 1 required positional argument: 'self'
2020-43-20 15:43:08-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-43-20 15:43:08-__init__-DEBUG-Using selector: SelectSelector
2020-43-20 15:43:11-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-43-20 15:43:11-__init__-DEBUG-Using selector: SelectSelector
2020-43-20 15:43:27-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-43-20 15:43:27-__init__-DEBUG-Using selector: SelectSelector
2020-43-20 15:43:30-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-43-20 15:43:30-__init__-DEBUG-Using selector: SelectSelector
2020-43-20 15:43:36-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-43-20 15:43:36-__init__-DEBUG-Using selector: SelectSelector
2020-47-20 15:47:34-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-47-20 15:47:34-__init__-DEBUG-Using selector: SelectSelector
2020-47-20 15:47:36-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-47-20 15:47:36-__init__-DEBUG-Using selector: SelectSelector
2020-47-20 15:47:42-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-47-20 15:47:42-__init__-DEBUG-Using selector: SelectSelector
2020-49-20 15:49:05-trigger_reload-INFO-D:\Study\School\e2e\server\server\settings.py changed, reloading.
2020-49-20 15:49:05-__init__-DEBUG-Using selector: SelectSelector
2020-50-20 15:50:32-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 128, in messageDetail
    loginingUserid = int(request.session["loginingUserid"])
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\contrib\sessions\backends\base.py", line 64, in __getitem__
    return self._session[key]
KeyError: 'loginingUserid'
2020-50-20 15:50:49-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-50-20 15:50:49-__init__-DEBUG-Using selector: SelectSelector
2020-50-20 15:50:51-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-50-20 15:50:51-__init__-DEBUG-Using selector: SelectSelector
2020-52-20 15:52:13-trigger_reload-INFO-D:\Study\School\e2e\server\server\settings.py changed, reloading.
2020-52-20 15:52:13-__init__-DEBUG-Using selector: SelectSelector
2020-52-20 15:52:46-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-52-20 15:52:46-__init__-DEBUG-Using selector: SelectSelector
2020-53-20 15:53:01-log_response-ERROR-Internal Server Error: /apis/message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\server\backend\views.py", line 128, in messageDetail
    loginingUserid = int(request.session.get("loginingUserid"))
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
2020-53-20 15:53:13-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-53-20 15:53:13-__init__-DEBUG-Using selector: SelectSelector
2020-53-20 15:53:19-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-53-20 15:53:19-__init__-DEBUG-Using selector: SelectSelector
2020-53-20 15:53:21-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-53-20 15:53:21-__init__-DEBUG-Using selector: SelectSelector
2020-53-20 15:53:50-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-53-20 15:53:50-__init__-DEBUG-Using selector: SelectSelector
2020-53-20 15:53:58-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-53-20 15:53:58-__init__-DEBUG-Using selector: SelectSelector
2020-55-20 15:55:07-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-55-20 15:55:07-__init__-DEBUG-Using selector: SelectSelector
2020-55-20 15:55:11-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-55-20 15:55:11-__init__-DEBUG-Using selector: SelectSelector
2020-55-20 15:55:13-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-55-20 15:55:13-__init__-DEBUG-Using selector: SelectSelector
2020-55-20 15:55:46-trigger_reload-INFO-D:\Study\School\e2e\server\server\settings.py changed, reloading.
2020-55-20 15:55:46-__init__-DEBUG-Using selector: SelectSelector
2020-55-20 15:55:51-trigger_reload-INFO-D:\Study\School\e2e\server\server\settings.py changed, reloading.
2020-55-20 15:55:51-__init__-DEBUG-Using selector: SelectSelector
2020-55-20 15:55:54-trigger_reload-INFO-D:\Study\School\e2e\server\server\settings.py changed, reloading.
2020-55-20 15:55:54-__init__-DEBUG-Using selector: SelectSelector
2020-55-20 15:55:57-trigger_reload-INFO-D:\Study\School\e2e\server\server\settings.py changed, reloading.
2020-55-20 15:55:57-__init__-DEBUG-Using selector: SelectSelector
2020-56-20 15:56:00-trigger_reload-INFO-D:\Study\School\e2e\server\server\settings.py changed, reloading.
2020-56-20 15:56:00-__init__-DEBUG-Using selector: SelectSelector
2020-56-20 15:56:33-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-56-20 15:56:33-__init__-DEBUG-Using selector: SelectSelector
2020-57-20 15:57:07-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-57-20 15:57:07-__init__-DEBUG-Using selector: SelectSelector
2020-57-20 15:57:28-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-57-20 15:57:28-__init__-DEBUG-Using selector: SelectSelector
2020-57-20 15:57:53-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-57-20 15:57:53-__init__-DEBUG-Using selector: SelectSelector
2020-58-20 15:58:12-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-58-20 15:58:12-__init__-DEBUG-Using selector: SelectSelector
2020-13-20 16:13:36-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-13-20 16:13:36-__init__-DEBUG-Using selector: SelectSelector
2020-13-20 16:13:44-trigger_reload-INFO-D:\Study\School\e2e\server\backend\views.py changed, reloading.
2020-13-20 16:13:44-__init__-DEBUG-Using selector: SelectSelector
2020-21-20 16:21:49-__init__-DEBUG-Using selector: SelectSelector
